src/content/docs/fr/2/guide/prerequisites/index.mdx

```diff
@@ -0,0 +1,315 @@
+
+### Linux
+
+Tauri exige un système varié de dépendances pour le développement sur Linux. Celles-ci peuvent être différentes selon votre distribution, mais nous avons inclus quelques distributions populaires ci-dessous pour vous aider durant l'installation.
+
+{/* Note: Ce sont les distributions supportées officielles. */}
+{/* Si vous voulez en ajouter une autre, ouvrez une issue sur github pour discuter avant d'ouvrir un PR */}
+
+<Tabs>
+  <TabItem label="Debian">
+
+```sh
+sudo apt update
+sudo apt install libwebkit2gtk-4.0-dev \
+  build-essential \
+  curl \
+  wget \
+  file \
+  libssl-dev \
+  libgtk-3-dev \
+  libayatana-appindicator3-dev \
+  librsvg2-dev
+```
+
+  </TabItem>
+  <TabItem label="Arch">
+
+```sh
+sudo pacman -Syu
+sudo pacman -S --needed \
+  webkit2gtk \
+  base-devel \
+  curl \
+  wget \
+  file \
+  openssl \
+  appmenu-gtk-module \
+  gtk3 \
+  libappindicator-gtk3 \
+  librsvg \
+  libvips
+```
+
+  </TabItem>
+  <TabItem label="Fedora">
+
+```sh
+sudo dnf check-update
+sudo dnf install webkit2gtk4.0-devel \
+  openssl-devel \
+  curl \
+  wget \
+  file \
+  libappindicator-gtk3-devel \
+  librsvg2-devel
+sudo dnf group install "C Development Tools and Libraries"
+```
+
+  </TabItem>
+  <TabItem label="Gentoo">
+
+```sh
+sudo emerge --ask \
+  net-libs/webkit-gtk:4.1 \
+  dev-libs/libappindicator \
+  net-misc/curl \
+  net-misc/wget \
+  sys-apps/file
+```
+
+  </TabItem>
+  <TabItem label="openSUSE">
+
+```sh
+sudo zypper up
+sudo zypper in webkit2gtk3-soup2-devel \
+  libopenssl-devel \
+  curl \
+  wget \
+  file \
+  libappindicator3-1 \
+  librsvg-devel
+sudo zypper in -t pattern devel_basis
+```
+
+  </TabItem>
+  <TabItem label="NixOS">
+
+TODO: Need to build out NixOS instructions
+
+  </TabItem>
+</Tabs>
+
+Si votre distribution n'est pas incluse ci-dessus, vous devriez jeter un œil ici: [Awesome Tauri on GitHub](https://github.com/tauri-apps/awesome-tauri#guides) pour voir si un guide a été créé.
+
+Suivant: [Installation de Rust](#rust)
+
+### macOS
+
+Tauri utilise une plusieurs dépendances macOS et iOS de [Xcode](https://developer.apple.com/xcode/resources/).
+
+Téléchargez et installez Xcode à partir des sites suivants :
+
+- [Mac App Store](https://apps.apple.com/gb/app/xcode/id497799835?mt=12)
+- [Apple Developer website](https://developer.apple.com/xcode/resources/).
+
+Assurez-vous de lancer Xcode après installation pour qu'il puisse finir l'installation.
+
+<details>
+<summary>Développement uniquement pour les applications sur ordinateur?</summary>
+Si vous planifiez de développer des applications de bureau qui ne prennent pas en charge iOS vous pouvez installer Xcode Command Line Tools à la place:
+
+```sh
+xcode-select --install
+```
+
+</details>
+
+Suivant: [Installation de Rust](#rust)
+
+### Windows
+
+Tauri utilise les Outils de Développement C++ de Microsoft (Microsoft C++ Build Tools) ainsi que Microsoft Edge WebView2. Ils sont tous les deux requis pour le développement Windows.

```

Comment:
I'll leave this to @lemarier to give a call on, but I think "les Outils de Développement C++ de Microsoft" should be kept to "Microsoft C++ Build Tools" since it's a proper noun? Similar to how the heading directly below is kept in English

Reply:
Makes sens, fixed